pytorch/training/docker/2.0/py3/Dockerfile.cpu

```diff
@@ -257,10 +250,12 @@ ARG PT_TORCHVISION_URL
 ARG PT_TORCHAUDIO_URL
 ARG PT_TORCHDATA_URL
 
-# Install AWS-PyTorch and other torch packages
-RUN pip uninstall -y torch torchvision torchaudio torchdata \
- && pip install --no-cache-dir -U ${PT_SM_TRAINING_URL} ${PT_TORCHVISION_URL} ${PT_TORCHAUDIO_URL} ${PT_TORCHDATA_URL}
-
+# force remove to prevent removing all packages depending on pytorch
```

Comment:
why do we need to remove pytorch and re-install aws-pytorch again here? didn't we already installed pytorch in line 131? https://github.com/aws/deep-learning-containers/pull/2742/files#diff-b652891554b7a0cffc9130c708e5dec5cc61d15e8105030951e45ba234a842fbR131

Reply:
This is for removing the ec2 package and installing the sm channel package. --force flags makes sure no packages that depend on pytorch are removed